doc/source/notebooks/intro.ipynb

```diff
@@ -13,38 +13,38 @@
     "  - In [gplvm.ipynb](basics/GPLVM.ipynb) we cover the unsupervised case, and showcase GPflow's Bayesian GPLVM implementation (Titsias and Lawrence, 2010).\n",
     "\n",
-    "In each of these notebooks we go over the data format, model setup, model optimisation and predict options.\n",
+    "In each of these notebooks we go over the data format, model setup, model optimisation and prediction options.\n",
     "\n",
     "## Understanding\n",
     "\n",
-    "This section covers the building blocks of GPflow from a implementation perspective, and show how the different modules interact as a whole.\n",
+    "This section covers the building blocks of GPflow from an implementation perspective, and shows how the different modules interact as a whole.\n",
     "<!-- - [Architecture](understanding/architecture.ipynb)  **[TODO]** -->\n",
-    "<!--  - [Utilities](understanding/utilities.ipynb): expectations, multi-output, conditionals, kullbach leibers (KL), log-densities, features and quadrature  **[TODO]** -->\n",
+    "<!--  - [Utilities](understanding/utilities.ipynb): expectations, multi-output, conditionals, Kullback-Leibler divergences (KL), log-densities, features and quadrature  **[TODO]** -->\n",
     "<!--  - [Manipulating models](understanding/models.ipynb)  **[TODO]**  -->\n",
     "  - [Handling TensorFlow graph and sessions](understanding/tf_graphs_and_sessions.ipynb).\n",
-    "  - [Tips and Tricks](tips_and_tricks.ipynb) a more elaborate list of how GPflow works togheter with TensorFlow.\n",
+    "  - [Tips and Tricks](tips_and_tricks.ipynb): a more elaborate list of how GPflow works together with TensorFlow.\n",
     "\n",
     "    \n",
     "## Advanced Needs\n",
     "\n",
     "GPflow also allows for more complex features and models:\n",
     "\n",
     "*Models:*\n",
-    "  - [MCMC](advanced/mcmc.ipynb) HMC to approximate the posterior GP and hyperparameters.\n",
+    "  - [MCMC](advanced/mcmc.ipynb) instead of variational inference: Hamiltonian Monte Carlo to sample the posterior GP and hyperparameters.\n",
     "  - [Ordinal regression](advanced/ordinal_regression.ipynb) shows how to use GPflow for dealing with ordinal variables.\n",
-    "  - [Multi-class classification](advanced/multiclass_classification.ipynb) classification in the non-binary case.\n",
-    "  - [Multi-output with coregionalisation](advanced/coregionalisation.ipynb)\n",
-    "  - [Multi-outputs with SVGPs](advanced/multioutput.ipynb)\n",
-    "  - When you're dealing with a large datasets (over 1k), you want to resort to Sparse methods. In [GPs for big data.ipynb](advanced/gps_for_big_data.ipynb)  we show how to use GPflow's Sparse Variational GP (SVGP) model (Hensman et al., 2013; 2015)\n",
+    "  - [Multi-class classification](advanced/multiclass_classification.ipynb): classification in the non-binary case.\n",
+    "  - [Multi-output with coregionalisation](advanced/coregionalisation.ipynb) for heterotopic observations.\n",
```

Comment:
I am not sure of the term heterotopic is well-known outside of the GP multi-output community, but I could be wrong!

Reply:
Pleeaaaase tell me what else to call it D:

Reply:
I like **fully observed outputs** for isotropic and **partially observed outputs** for hetrotopic. Although, I can see where this can get confusing...